javascript/ql/src/Security/CWE-200/PrivateFileExposure.ql

```diff
@@ -80,10 +80,14 @@ Folder getAPackageJSONFolder() { result = any(PackageJSON json).getFile().getPar
 DataFlow::Node getALeakingFolder(string description) {
   exists(ModuleScope ms | result.asExpr() = ms.getVariable("__dirname").getAnAccess()) and
   result.getFile().getParentContainer() = getAPackageJSONFolder() and
-  description = "the folder " + result.getFile().getParentContainer().getRelativePath()
+  (
+    if result.getFile().getParentContainer().getRelativePath().trim() != ""
+    then description = "the folder " + result.getFile().getParentContainer().getRelativePath()
+    else description = "a folder"
```

Comment:
> Serves a folder, which can contain private information.

So the comma above is kind of unfortunate as it suggests that folders in general can contain private information.

Could we reasonably change this to `the source root folder`?